.github/workflows/fossa.yml

```
@@ -12,6 +14,12 @@ jobs:
       - name: checkout code
         uses: actions/checkout@v2
       - name: Run FOSSA scan and upload build data
-        uses: fossa-contrib/fossa-action@v1
-        with:
-          fossa-api-key: 7ac0174603e25cb0a4b3ebd2e131a89f
+        #uses: fossa-contrib/fossa-action@v1
```

```suggestion
        uses: fossas/fossa-action@v1
```
Try this one. Seems this is the official fossa-action.

---

works well

---

Nice.